.github/workflows/actions.yml

```diff
@@ -34,6 +24,36 @@ jobs:
         with:
           options: --check --diff
           src: .
+      - name: Install flake8, isort
+        run: pip install flake8 isort
+      - name: Lint (flake8)
+        run: flake8
+      - name: Lint (isort)
+        run: isort --check .
+      - name: Start PostgreSQL DB
+        run: docker-compose run --detach db
```

Comment:
After this step, the docker-compose-managed PostgreSQL container is left running, available for subsequent steps. Effectively replacing
```
    services:
      postgres:
        image: postgres:11
```